community/portworx/templates/test/test-portworx-api.yaml

```
@@ -0,0 +1,30 @@
+{{- $registrySecret := .Values.registrySecret | default "none" }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-api-test-{{ randAlphaNum 5 | lower }}"
+  namespace: kube-system
+  labels:
+{{- include "px.labels" . | nindent 4 }}
```

Should add a unique label for this pod.  Maybe:

```suggestion
    name: test-pod
{{- include "px.labels" . | nindent 4 }}
```